pom.xml

```diff
@@ -131,6 +131,7 @@
                 <artifactId>rascal-maven-plugin</artifactId>
                 <version>${rascal-maven.version}</version>
                 <configuration>
+                    <rascalBootstrapVersion>0.40.1-RC21</rascalBootstrapVersion>
```

Comment:
Similar here?

Reply:
0.41.0-RC21 is the first version that I hope will be able to generate .tpl files using the commandline version of the checker/compiler. I have turned off still the tutor and the packager, because I needed some bugfixes there.